.travis.yml

```diff
@@ -32,23 +32,18 @@ jobs:
         - make .gitvalidation
         - make gofmt
         - make lint
+      go: tip
+    - stage: Build and Verify
+      script:
         - make testunit
-        - make docs
         - make
       go: 1.8.x
-    - stage: Build and Verify
-      script:
-        - make .gitvalidation
-        - make gofmt
-        - make lint
+    - script:
```

Comment:
Is this right?  You now have [two “Build and Verify” runs with Go tip][1].  How about:

```yaml
jobs:
  include:
    - stage: Lint
      script:
        - make .gitvalidation
        - make gofmt
        - make lint
      go: tip
    - stage: Build and Verify
      script:
        - make testunit
        - make
      go: 1.8.x
    - stage: Build and Verify
      script:
        - make testunit
        - make
      go: 1.9.x
    - stage: Build and Verify
      script:
        - make testunit
        - make
      go: tip
    - stage: Integration Test
      script:
        - make integration
      go: 1.9.x
```

[1]: https://travis-ci.org/kubernetes-incubator/cri-o/builds/348420315?utm_source=github_status&utm_medium=notification

Reply:
I thought about it, the problem is that if they are in the same stage they can run in parallel, but the stages go one after another.

I used `go tip` for `.gitvalidation, gofmt & lint` but if you see the other `go tip` section is for `testunit & make`. Maybe I could change the substage description somehow to don't show `go tip` twice in Travis.

Reply:
> I thought about it, the problem is that if they are in the same stage they can run in parallel, but the stages go one after another.

Ah.  I'm ok blocking the unit and build tests until the lint stage finishes, but I don't really mind.

Reply:
In that case the reorganization of the Travis files wouldn't be done for speed purposes [1] but it would be more clear which is good as well. I am happy with any of the ways.

[1] To be honest, I thought that removing duplicated steps the speed would increase, but most of the time is spent in `before_install` so the difference is minimal.